integration_tests/tests/apply/about_the_person/equality_and_diversity/complete_disability_page.cy.ts

```diff
@@ -0,0 +1,113 @@
+//  Feature: Referrer completes disability question page
+//    So that I can complete the 'Equality questions' task
+//    As a referrer
+//    I want to answer questions on the disability page
+//
+//  Scenario: answer is enforced
+//    Given I'm on the 'Will answer equality questions?' task page
+//    When I try to continue without answering the question
+//    Then I see that an answer is required
+//
+//  Scenario: submit 'other' disability type
+//    Given I'm on the 'Will answer equality questions?' task page
+//    When I submit an 'other' disability type
+//    Then I return to the task list page
+//    And I see that the task has been completed
```

Comment:
I think we mean to say "Disability page" rather than "Will answer equality questions" page?